common-rendering/src/newsletterEmbedIframe.ts

```diff
@@ -0,0 +1,56 @@
+type NewsletterHeightEventType = { source: object; origin: string };
```

Comment:
Can we perhaps keep this code local to DCR and simply duplicate it in AR, rather than using common-rendering?

I don't think this sort of code belongs here. It was conceived for sharing components but it's unfortunately now very inviting as a place to put things but perhaps we shouldn't be so quick to share and having the sorts of checks and balances you get with things like `@guardian/libs` is arguably a good thing

For example, I am already aware that we have at least two, possibly three, different ways of resizing iframes in DCR so there's clearly a need to review this and consider an abstraction. But that kind of refactoring work is harder when you've already got coupled code.

Reply:
I'm inclined to agree here, especially in the context of what `common-rendering` sets out to achieve. I will move this utility back into AR/DCR - which is essentially how things were.